### ISSUE TITLE
Bind to hasError property of the form

### ISSUE THREAD

**Issue by peterPgk** [matfish2/vue-formular#13]
Great plugin, I use it with pleasure :)
Can you help me with one problem? 
Is there a event which fires, when form **hasErrors** property is changed? I want to bind reactively to this property, so I can change accordingly one property of the parent element.

I have this structure:

```
<parent>
    <child>
    <vue-formular>
```

To achieve this I tried to create computed property in parent element

```
computed: {
    isValid: !this.$refs.form.hasErrors
}
```

but recieve this error : 

`Cannot read property '$refs' of undefined`

When I do this with getter funtion it seems to work, until I try to bind this property to other child element

`<child :valid="isValid">`

and then this error is coming:

```
[Vue warn]: Error when evaluating expression "function get() {
    return this.$refs.form.hasErrors;
} Cannot read property 'hasErrors' of undefined
```

I tried to use watcher insteed, but with same errors.

Is it posible to bind to hasError property?

**Comment by matfish2** [comment]
computed properties should be functions, so instead of

``` js
computed: {
    isValid: !this.$refs.form.hasErrors
}
```

try

``` js
computed: {
    isValid: function() {
      return !this.$refs.form.hasErrors
  }
}
```

**Comment by peterPgk** [comment]
Thank you for your quick support.
This with function is working, but again, when I try to bind **isValid** to other child component, receive this error while still form is initializing:

```
vue.common.js?4a36:1019[Vue warn]: Error when evaluating expression "function isValid() {
            return !this.$refs.form.hasErrors;
        }": TypeError: Cannot read property 'hasErrors' of undefined
```

**Comment by matfish2** [comment]
Add an existence check:

``` js
computed: {
    isValid: function() {
      return this.$refs.form?!this.$refs.form.hasErrors:true;
  }
}
```

**Comment by peterPgk** [comment]
Thank you,
I found a similar solution too and it works, but because I use slots to put components inside, the problem continues. 
Probably the problem is in slots - I found this [https://github.com/vuejs/vue/issues/2494](https://github.com/vuejs/vue/issues/2494) with explanation that they are a bit limited.

Thank you again for your quick help, I will try to construct my components without using slots.
